thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js

```diff
@@ -219,6 +231,31 @@ export default Ember.Component.extend({
     }
   },
 
+  _transformSeries(mode, series) {
+    switch(mode) {
+      case 'absolute':
```

Comment:
let's save the string `absolute`, `relative` and `log` into a const obj TYPE
and refer to them as TYPE.absolute, TYPE.relative and TYPE.log

https://github.com/linkedin/pinot/pull/2081/files#diff-ba88a3a55f4254f597817fd4301d5280R20 can also be updated accordingly

Reply:
fixed